app/components/hero/campaignHeroSubText.tsx

```diff
@@ -8,7 +8,18 @@ import {
 } from "@nypl/design-system-react-components";
 import React from "react";
 
-const CampaignHeroSubText = ({ featuredItem }: any) => {
+interface CampaignHeroSubtextProps {
+  featuredItem: {
```

Comment:
I was going to say to use `FeaturedItemData` but that one contains `numberOfDigitizedItems`. This feels more correct and perhaps what's in `app/types/FeaturedItemData/FeaturedItemData.ts` should be renamed to be slightly more generic.

Reply:
Maybe `HeroData`? I do find it a little confusing that we have `FeaturedItem` everywhere and then at some point you have to clock that that's actually the data going into the `CampaignHero` component, it'd be nice to have it be consistent everywhere. What do you think @avertrees

Reply:
`HeroData` sounds good. We can work on that next week.

Reply:
Now that this has been updated with `main`, should the name be updated? @avertrees